compaction/compaction_manager.cc

```diff
@@ -462,8 +467,7 @@ class sstables_task_executor : public compaction_task_executor, public sstables_
         set_sstables(std::move(sstables));
     }
 
-    virtual ~sstables_task_executor();
```

Comment:
please keep the virtual destructor for future compatibility so that derived classes can safely define their own.